manim/animation/transform.py

```diff
@@ -240,6 +297,63 @@ def __init__(
 
 
 class CounterclockwiseTransform(Transform):
+    """Transforms a mobject by moving each of its points along an arc with
```

Comment:
```suggestion
    """Transforms an mobject along an arc while interpolating it counterclockwise
```
Again, a concise description